### PR TITLE
Bug 1268539 - Add a script to simplify the standalone and release to m-c processes.

### PR DESCRIPTION
**Pull request by Standard8**

### CODE REVIEW

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: bin/create_release.py [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+    return result
+
+
+def releaseStandalone(bugzilla, repo, useSystem, version, username):
+    print "Creating a bug for standalone"
+
+    print "Please enter the previous release version:"
+    previousRelease = raw_input().rstrip("\r\n")
+
+    print "Please enter required timescales, e.g.:"
```
Comment (dmose) [inline bin/create_release.py]: Calling it a timescale seems a bit confusing.  Maybe just "Please enter any requests for the deployers, e.g.:"?

File: bin/create_release.py [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+                        help="Bugzilla API key.",
+                        required=True)
+    parser.add_argument("--rel-version",
+                        metavar="1.2.1",
+                        help="The version number to release, e.g. '1.2.1'.",
+                        required=True)
+    parser.add_argument("--mozilla-central-repo",
+                        metavar="../gecko-dev",
+                        help="A gecko directory reference to mozilla-central")
+    parser.add_argument("--irc-nick",
```
Comment (dmose) [inline bin/create_release.py]: Maybe this wants to be --reviewer-irc-nick for clarity?

**Comment by dmose** [comment]
r=dmose, though I didn't go so far as to actually test this myself.

**Pull request merged**